More properties do not accept number

ewilligers opened the pull request:
Test that various additional properties do not accept <number>
in quirks mode.

ewilligers commented:
These tests cover the red cells in 
https://docs.google.com/spreadsheets/d/1HRzb_-S28xq7GqYKhHbMP2YQlqWLvOWSS_twLXy-I40/edit#gid=1297517027

The yellow cells are being discussed in the CSS or SVG working groups.

zcorpan commented on quirks/unitless-length/excluded-properties-002.html:
```diff
@@ -0,0 +1,41 @@
+
+for (let property of properties) {
+  test(() => {
+    if (!getComputedStyle(target)[property])
```
I think this should fail the test instead, e.g. using assert_unreached

ewilligers replied:
Now fails the test if the property is not known.

ericwilligers committed:
More properties do not accept number

Test that various additional properties do not accept <number>
in quirks mode.

The tests fail if properties are not supported.
